contrib/gofiber/fiber.v2/option.go

```diff
@@ -82,6 +88,31 @@ func WithStatusCheck(fn func(statusCode int) bool) Option {
 	}
 }
 
+// WithResourceNamer specifies a function which will be used to
+// obtain the resource name for a given request taking the go-fiber context
+// as input
+func WithResourceNamer(fn func(*fiber.Ctx) string) Option {
+	return func(cfg *config) {
+		cfg.resourceNamer = fn
+	}
+}
+
+// WithPathParams specifies that the integration should attach route
+// parameters as tags to spans.
+//
+// Warning: using this feature can risk exposing sensitive data
+func WithPathParams() Option {
+	return func(cfg *config) {
+		cfg.pathParams = true
+	}
+}
+
+func defaultResourceNamer(c *fiber.Ctx) string {
+	route := c.Route()
+
+	return fmt.Sprint(route.Method, " ", route.Path)
```

Comment:
```suggestion
	r := c.Route()
	return r.Method + " " + r.Path
```
Let's avoid using `fmt.Sprint`. It is quite slow.